markdown/bitburner.charityeventgendata.modifiers.md

```diff
@@ -0,0 +1,11 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [bitburner](./bitburner.md) &gt; [CharityEventGenData](./bitburner.charityeventgendata.md) &gt; [modifiers](./bitburner.charityeventgendata.modifiers.md)
+
+## CharityEventGenData.modifiers property
+
+**Signature:**
+
+```typescript
+modifiers: modifiers[];
```

Comment:
`modifiers` should likely be a `Modifier[]` (capitalized and singular), although if Modifier is a type specific to Charity then it should be named something more unique.